trie/trie.go

```diff
@@ -69,24 +69,25 @@ type (
 		numBranch uint64
 		numExt    uint64
 		numLeaf   uint64
-		dao       db.CachedKVStore
+		cb        db.CachedBatch // cached batch for pending writes
+		dao       db.KVStore     // the underlying storage DB
```

Comment:
don't call it dao, state factory is a kind of DAO though not typical. maybe call it store/db